packages/docs/stories/templates/form.stories.tsx

```diff
@@ -0,0 +1,182 @@
+     * We just let them flow automatically mobile and arrange them
+     * in two columns when space is wide enough
+     */
+    syn-radio-group::part(form-control-input) {
+      display: grid;
+    }
+
+    @media screen and (min-width: 641px) {
+      syn-radio-group::part(form-control-input) {
+        grid-auto-flow: column;
+        grid-template-rows: 1fr 1fr 1fr;
+      }
+    }
+
+    .submit-actions {
+      display: flex;
+      justify-content: right;
+      margin-bottom: var(--syn-font-size-x-large);
+    }
+    </style>
+    <div class="synergy-form-demo">
+      <h1>Kontaktformular</h1>
+      <p>Bitte geben Sie Ihre Kontaktdaten an und beschreiben Sie, wobei wir Sie unterstützen können.</p>
+
+      <form method="post" id="syn-form-demo">
+        
+        <fieldset>
+          <syn-radio-group
+            label="Zu welchem Thema haben Sie Fragen?"
+            name="topic"
+            required
+          >
+            <syn-radio value="1">Anfrage/Angebot</syn-radio>
+            <syn-radio value="2">Aufträge/Rechnungen</syn-radio>
+            <syn-radio value="3">Rücksendungen/Reklamation</syn-radio>
+            <syn-radio value="4">Zubehörauswahl</syn-radio>
+            <syn-radio value="5">Applikationserklärung</syn-radio>
+            <syn-radio value="6">Inbetriebnahmeunterstützung</syn-radio>
+          </syn-radio-group>
+        </fieldset>
+
+        <fieldset>
+          <legend>Beschreiben Sie Ihre Anfrage</legend>
+          <p>
+            Es ist sehr hilfreich, wenn die Beschreibung möglichst genau ist, so können wir noch spezieller auf Ihre Fragen eingehen.
+            (Bei Applikationsbeschreibungen ggf. Material/Abmessungen/Geschwindigkeiten mit angeben).
+          </p>
+
+          <syn-textarea label="Ihre Nachricht" name="message" required></syn-textarea>
+        </fieldset>
+
+        <fieldset>
+          <legend>Ihre Kontaktdaten:</legend>
+          <div class="fields">
+            <syn-input label="Kundennummer" name="customerNr"></syn-input>
+            <syn-input label="Firmenname" name="companyName" required></syn-input>
+            <syn-input label="Adresse" name="address" required></syn-input>
+            <syn-input label="PLZ" name="zip" required></syn-input>
+            <syn-input label="Ort" name="city" required></syn-input>
+            <syn-input label="Land" name="country" required></syn-input>
+            <syn-input label="Ansprechpartner" name="salesPerson" required></syn-input>
+            <syn-input type="tel" label="Telefonnummer" name="phone"></syn-input>
+            <syn-input type="tel" label="Fax" name="fax"></syn-input>
+            <syn-input type="email" label="E-Mail-Adresse" name="mail" required></syn-input>
+          </div>
+          <syn-checkbox name="subscribeNewsletter">
+            Ja, ich möchte regelmäßig aktuelle und interessanteInformationen zu Lösungen mit Produkten, Systemenund Services von SICK per E-Mail erhalten.
```

Comment:
Typos. But I'm not sure if this should already be fixed now or if we wait for the data from design. :)